src/ClientTrait.php

```diff
@@ -36,7 +36,7 @@
      * @param ConnectionInterface $connection Represents a connection to a Google
      * Cloud Platform Service.
      */
-    public function setConnection(ConnectionInterface $connection)
+    public function setConnection($connection)
```

Comment:
This is also removed. Assuming #36 is merged first, I'll update my branch to remove this stuff.